app/Resources/views/easy_admin/full_limited_show.html.twig

```diff
@@ -0,0 +1,5 @@
+{% if value == true %}
```

Comment:
If `value` is a boolean this should be `{% if value %}`. Otherwise it should be `{% if value same as true %}` (cf. https://twig.symfony.com/doc/2.x/tests/sameas.html).

Reply:
c/p from https://github.com/itk-event-database/event-database-api/blob/develop/app/Resources/views/easy_admin/yes_no.html.twig but happy to fix ;-)